test/JDBC/expected/single_db/db_accessadmin-vu-verify.out

```diff
@@ -0,0 +1,337 @@
+
+-- MEMBERS OF db_accessadmin WILL ALWAYS HAVE CONNECT PRIVILEGES
+REVOKE CONNECT FROM babel_5136_db_accessadmin_user
+GO
+-- terminate-tsql-conn
+
+-- tsql user=babel_5136_l1 password=12345678 database=babel_5136
+-- Only db_owner members should be able to do this
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_db_accessadmin_user
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the role 'db_accessadmin', because it does not exist or you do not have permission.)~~
+
+-- terminate-tsql-conn
+
+
+-- tsql user=babel_5136_db_accessadmin_l1 password=12345678
+-- Check all allowed operations
+USE babel_5136
+GO
+
+SELECT current_user
+GO
+~~START~~
+varchar
+babel_5136_db_accessadmin_user
+~~END~~
+
+
+-- should be a member of db_accessadmin in babel_5136 database
+SELECT IS_ROLEMEMBER('db_accessadmin')
+GO
+~~START~~
+int
+1
+~~END~~
+
+
+-- CREATE DROP A NEW USER
+CREATE USER babel_5136_u2 FOR LOGIN babel_5136_l2
+GO
+DROP USER babel_5136_u2
+GO
+
+-- DROP & RECREATE EXISTING USER
+DROP USER babel_5136_u1
+GO
+CREATE USER babel_5136_u1 FOR LOGIN babel_5136_l1
+GO
+
+-- ALTER EXISTING USER and THEN DROP IT
+ALTER USER babel_5136_u1 WITH DEFAULT_SCHEMA = dbo
+GO
+ALTER USER babel_5136_u1 WITH NAME = babel_5136_u1_new_name
+GO
+-- NOT SUPPORTED CURRENTLY IN BABELFISH
+ALTER USER babel_5136_u1_new_name WITH PASSWORD = 'shouldfail'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near '<EOF>' at line 3 and character position 0)~~
+
+DROP USER babel_5136_u1_new_name
+GO
+CREATE USER babel_5136_u1_new_name FOR LOGIN babel_5136_l1
+GO
+-- ALTER NEW USER and THEN DROP IT
+CREATE USER babel_5136_u2 FOR LOGIN babel_5136_l2
+GO
+ALTER USER babel_5136_u2 WITH NAME = babel_5136_u1
+GO
+
+-- CREATE SCHEMA should be allowed db_accessadmin
+CREATE SCHEMA s1
+GO
+CREATE SCHEMA s2 AUTHORIZATION dbo
+GO
+CREATE SCHEMA s3 AUTHORIZATION babel_5136_u1
+GO
+-- Should only be able to drop schema that it owns
+DROP SCHEMA s1
+GO
+DROP SCHEMA s2
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: must be owner of schema s2)~~
+
+DROP SCHEMA s3
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: must be owner of schema s3)~~
+
+
+-- Should be restricted
+DROP USER dbo
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the user 'dbo'.)~~
+
+DROP ROLE db_owner
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the role 'db_owner'.)~~
+
+DROP ROLE db_accessadmin
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the role 'db_accessadmin'.)~~
+
+DROP USER guest
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: User 'guest' cannot be dropped, it can only be disabled. The user is already disabled in the current database.)~~
+
+
+-- db_accessadmin can not add/drop members to any other role
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the role 'db_accessadmin', because it does not exist or you do not have permission.)~~
```

Comment:
Please add testcases for
1. ALTER ROLE db_accessadmin ADD MEMBER [any other fixed role -- for now sysadmin & db_owner]
2. ALTER ROLE db_accessadmin DROP MEMBER
3. sp_addrolemember
4. sp_droprolemember

Reply:
Added